### ISSUE TITLE
Workspaces directory not working

### ISSUE THREAD

**Issue by inmanturbo** [paperwm/PaperWM#595]
**Describe the bug**
The workspaces directory thingy is a great idea doesn't seem to work? i set a directory but it has no effect on terminals, etc. Maybe I'm missing something?

**To Reproduce**
1. right click a workspace name and edit preferences
2. add a directory
3. open terminal
5. terminal won't open in workspace directory

**Expected behavior**
terminal should open in workspace directory

**System information:**
Please execute `./gather-system-info.sh` in you PaperWM clone and paste the output below.

```
Please include this information in your bug report on GitHub!
Distribution: Ubuntu
GNOME Shell 42.9
Display server: Xorg
PaperWM branch/tag: release
PaperWM commit: 04871fc0b47e3ab385ae7c856d332bbeca2b09b3
Enabled extensions:
- paperwm@paperwm.github.com
- gsconnect@andyholmes.github.io
- ding@rastersoft.com
- ubuntu-appindicators@ubuntu.com
- ubuntu-dock@ubuntu.com

```

**Additional context**

**Comment by inmanturbo** [comment]
> Note:     
It doesn't seem to matter how I open the terminal. Tried:
- `Ctrl+Alt+t`
- From the dock
- From the "show applications" grid

**Comment by Lythenas** [comment]
I haven't actually used that feature before.

I think the code for it is here: https://github.com/paperwm/PaperWM/blob/04871fc0b47e3ab385ae7c856d332bbeca2b09b3/app.js#L76

I don't fully grasp the code. But I believe it is at least supposed to work when you click one of the actions (e.g. "New Window") in the dropdown when clicking on the App name in the top bar. But I believe it was also supposed to just generally work when launching any new app.

For I also can't get it work at all on Gnome 44.4.

**Comment by jtaala** [comment]
Yes this is an older (but good) feature that has been broken for a while.

Only way at the moment (until it's fixed) is with `customHandlers` in `user.js` - see [this thread for it recently being used for emacs](https://paperwm.zulipchat.com/#narrow/stream/174096-general/topic/.E2.9C.94.20How.20to.20set.20customHandlers.20now.3F).
